cdn-ui/index.html

```diff
@@ -121,6 +128,7 @@ <h1>Clever Components smart CDN (WIP)</h1>
 import semver from 'https://jspm.dev/es-semver@1';
 
 const FIRST_CDN_VERSION = '5.3.1';
+let manifest_version = 0;
```

Comment:
should I avoid the global variable and pass this from function to function instead ?

Reply:
I don't like this design very much but here it's ok ;)

Reply:
Same, it's OK for now ;-)